ext/pdo/pdo_dbh.stub.php

```diff
@@ -40,7 +40,7 @@ public function prepare(string $statement, array $driver_options = []) {}
     public function query(string $statement, ?int $fetch_mode = null, mixed ...$fetch_mode_args) {}
```

Comment:
I think `$query` should be used instead of `$statement` (in `prepare()` too above). Couldn't be `$fetch_mode` 
 and `$fetch_mode_args` changed to just `$mode` and `$args`?